translation/README.md

```diff
@@ -0,0 +1,42 @@
+# Translation Process and Recommendations
+
+If you would like to contribute to the InnerSource Patterns translation, great! We would love to work with you!
+First, please refer to [CONTRIBUTING.md](/CONTRIBUTING.md) for basic contribution instructions.
+This document covers basic translation principles, how you can contribute, and the review process. Please note that the content of this document is recommendations, mainly aimed at removing barriers for new contributors to the translation process. If you see something missing, please feel free to make suggestions.
+
+## The Translation Principles and Recommendation
+
+- The target of the translation is structured patterns. You can find them at ```/patterns/2-structured/```, and only structured patterns are available in Gitbook.
```

Comment:
```suggestion
- The target of the translation are structured patterns. Find them in folder [/patterns/2-structured/](../patterns/2-structured/). Only structured patterns will be published in Gitbook.
```

Linking to the folder might make it even more clear which patterns we are talking about.

Also did some other rewording.

Reply:
LGTM👍